helper/schema/resource_data.go

```diff
@@ -213,9 +212,11 @@ func (d *ResourceData) State() *terraform.InstanceState {
 		}
 
 		raw := d.get([]string{k}, source)
-		rawMap[k] = raw.Value
-		if raw.ValueProcessed != nil {
-			rawMap[k] = raw.ValueProcessed
+		if raw.Exists && !raw.Computed {
```

Comment:
@mitchellh just did a small test (only the one that explicitly failed earlier) with this patch, but now the result of the computed value for `display_text` is also not saved anymore. I would say that's not correct and it's a backwards incompatibility with the previous behaviour...

And to make a stronger point for having those computed values written to the state, half the CloudStack provider dependants on the ability to save an read back computed values from the state consistently.

Will have some more tine in about 45 minutes to have a closer look as well...